src/pages/MainPage.js

```diff
@@ -1,10 +1,103 @@
+import styled from '@emotion/styled'
 import { Header, Banner } from '@components'
+import { useState, useEffect } from 'react'
+import { getChannelList, getPostListInChannel, getChannelInfo, getSearchedBookList } from '@apis'
+import { BookListSlider, Input, Button, Select } from '@components'
+
+const ALL_CATEGORY = 'all'
+const DEFAULT_CATEGORY = '소설'
+
+const SearchBar = styled.div`
+  display: flex;
+  justify-content: center;
+`
 
 const MainPage = () => {
+  const [postList, setPostList] = useState([])
+  const [categoryName, setCategoryName] = useState(ALL_CATEGORY)
+
+  const [searchKeyword, setSearchKeyword] = useState('')
+
+  const fetchAllPost = async () => {
```

Comment:
```suggestion
  const getAllPost = async () => {
```
`fetch`보다는 `get`을 사용하는 것이 직관적이기 때문에 바꾸는 것이 어떨까요??
밑에 다른 api 함수명도 수정하면 좋을 것 같습니다!